### PR TITLE
GEODE-5889 deploy_meta improvements

### PR DESCRIPTION
**Pull request by onichols-pivotal**
* explain how to install gcloud, if it isn't already installed
* work around concourse hiccups like reg/orange/brown jobs by retriggering them (once per invocation of deploy_meta)
* ensure window image is ready before starting windows jobs for the first time
* print nice progress indicators rather than spewing pages of debug while waiting for jobs
* display the correct URL when the pipeline is up
* don't pause golden reaper, and don't pause already-unpaused jobs on re-deploy

Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [x] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [x] Is your initial contribution a single, squashed commit?

- [n/a] Does `gradlew build` run cleanly?

- [n/a] Have you written or updated unit tests to verify your changes?

- [n/a] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

**Comment by onichols-pivotal** [comment]
@sgoller does this look good to you?

File: ci/pipelines/meta/deploy_meta.sh [hunk 1]
```diff
@@ -35,7 +35,21 @@ for cmd in Jinja2 PyYAML; do
 done
 
 if [ -z $(command -v gcloud) ]; then
-  echo "Install gcloud"
+  echo "Install gcloud (try brew cask install google-cloud-sdk, or see https://cloud.google.com/sdk/docs/downloads-interactive#mac)"
```
Comment (smgoller) [inline ci/pipelines/meta/deploy_meta.sh]: This part is fine. We could use this.

File: ci/pipelines/meta/deploy_meta.sh [hunk 2]
```diff
@@ -35,7 +35,21 @@ for cmd in Jinja2 PyYAML; do
...
+  exit 1
+fi
+
+forks=$(git remote -v | sed -e 's/.*github.com.//' -e 's#/.*##' | sort -u)
```
Comment (smgoller) [inline ci/pipelines/meta/deploy_meta.sh]: This part is not really useable at this point with incoming changes to deploy_meta.sh, we're shifting to a properties-style file of bash environment variables with local overrides. In that world, you create "meta.properties.local" and set GEODE_FORK to your upstream once and then you don't have to worry about it after that.

File: ci/pipelines/meta/deploy_meta.sh [hunk 3]
```diff
@@ -49,7 +63,6 @@ fi
 set -e
 set -x
 
-GEODE_FORK=${1:-"apache"}
```
Comment (smgoller) [inline ci/pipelines/meta/deploy_meta.sh]: All this gets pulled out into sourced environment variables files.

File: ci/pipelines/meta/deploy_meta.sh [hunk 4]
```diff
@@ -105,25 +118,12 @@ if [[ "$(tail -n1 flyOutput.log)" == "bailing out" ]]; then
   exit 1
 fi
 
-if [[ "${GEODE_FORK}" != "${UPSTREAM_FORK}" ]]; then
```
Comment (smgoller) [inline ci/pipelines/meta/deploy_meta.sh]: If everything from here down is just improving the post-deploy watching, that's fine.

**Pull request merged**